.env

```diff
@@ -0,0 +1 @@
+REACT_APP_BASE=https://176.241.137.67:18443
```

Comment:
@chooidos 
https://create-react-app.dev/docs/adding-custom-environment-variables/#adding-development-environment-variables-in-env
https://tacomanator.medium.com/environments-with-create-react-app-7b645312c09d
create an `.env.local` to override with `http://192.168.88.150:8080`, as it's your local network

Reply:
I think it should be changeable from application

Reply:
why? is it supposed to connect to different backends during its lifetime? Pretty sure we can statically assign different envs that won't need to be changed

Reply:
it will be static in my local network, but ISP could change external IP address.

Reply:
also, is it safe to store IP addresses in repo?

Reply:
got it
yes, good point, it's not very safe. Can you safely delete it?
https://docs.github.com/en/github/authenticating-to-github/keeping-your-account-and-data-secure/removing-sensitive-data-from-a-repository